frontend/public/style/_common.scss

```diff
@@ -399,29 +401,38 @@
   max-width: 600px;
 }
 
-.co-m-modal-link {
+// Enable word-break and append pficon-var-edit icon ::after so that it doesn't wrap without text
+.co-modal-btn-link {
+  box-sizing: content-box;
   cursor: pointer;
   outline: none;
-  padding-right:  20px;
-  position: relative;
+  padding: 0 20px 0 0;
+  text-align: left;
+  white-space: normal;
+  word-break: break-all; // Firefox
+  word-break: break-word;
   &::after {
     color: $color-pf-black-600;
     content: $pficon-var-edit;
+    display: inline-block;
     font-family: $icon-font-name-pf;
     line-height: 1;
+    margin-left: 5px;
+    margin-right: -20px; // width + margin-left
     pointer-events: none;
-    position: absolute;
+    position: relative;
     right: 0;
-    top: 0;
-  }
-  &.btn-link::after {
-    top: 5px;
+    width: 15px;
   }
   &:hover::after {
     color: $color-pf-black-700;
   }
 }
 
+.co-modal-btn-link--centered {
```

Comment:
Since `.btn` centers the text already, I suggest making the modifier `--left` and adding it to the cases that should *not* be centered.